hw/xbox/nv2a.c

```diff
@@ -3323,6 +3719,8 @@ static void pgraph_init(NV2AState *d)
     glextensions_init();
 
     assert(glo_check_extension("GL_EXT_texture_compression_s3tc"));
+    assert(glo_check_extension("GL_EXT_texture_snorm"));
+    assert(glo_check_extension("GL_ARB_ES2_compatibility"));
```

Comment:
GL_RGB_565 because we used to store 565 in 555 which might not be enough for some applications.